stacker/commands/stacker/diff.py

```diff
@@ -0,0 +1,35 @@
+""" Diffs the config against the currently running CloudFormation stacks
+
+Sometimes small changes can have big impacts.  Run 'stacker diff' before
+'stacker build' to detect bad things(tm) from happening in advance!
+"""
+
+from .base import BaseCommand
+from ...actions import diff
+
+
+class Diff(BaseCommand):
+    name = 'diff'
+    description = __doc__
+
+    def add_arguments(self, parser):
+        super(Diff, self).add_arguments(parser)
+        parser.add_argument("--force", action="append", default=[],
```

Comment:
Yeah, kinda.  Honestly, I like the fact that diff and build take the same args so once I've checked everything is good in my diff, I just need to edit the command on the cli and hit enter... less chance of making Bob unhappy. :)